File: 0001 Code Chef/0022 Fitness/implementation/0010 nodejs/fitness.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input)=>{
    let test = parseInt(input);
    let count = 0;

    rl.on('line',(input)=>{
        let distance = parseInt(input);
        let distance_walk_per_day = distance * 2;
        let distance_walk_per_week = distance_walk_per_day * 5;
        console.log(distance_walk_per_week);
        count = count + 1;
        if(count == test){
            rl.close();
        }
    })
})

File: 0001 Code Chef/0022 Fitness/implementation/0008 python3/fitness.py
if __name__ == "__main__":
    test  = int(input())
    while(test > 0):
        distance = int(input())
        distance_walk_per_day = 2*distance
        distance_walk_per_week = 5 * distance_walk_per_day
        print(distance_walk_per_week)
        test = test - 1